src/lib_rules.ml

```diff
@@ -377,12 +413,16 @@ module Gen (P : Install_rules.Params) = struct
          else
            acc)
      in
+     let deprecated_modules = Module.Name.Map.values wrapped_compat in
+     (* deprecated modules have implementations so we can just append them *)
```

Comment:
This comments is not clear to me. The reason is: nothing depends on compatibility modules and compatibility modules depends only on real modules

Reply:
Actually, my comment referred to the earlier line where we filter only the modules which have implementations. You're right that I should comment about the order as well.